src/components/Comments/ListComment.js

```diff
@@ -49,6 +51,7 @@ const ListComment = ({
         </View>
         {item?.children_counts?.comment > 0 && (
           <ReplyComment
+            feedId={feedId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `feedId` as a prop to the `ReplyComment` component is a good change, but make sure that the `ReplyComment` component is updated to handle this new prop and that all instances where the `ReplyComment` component is used are updated to pass this new prop.
```diff
+  feedId={feedId}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->